sematic/ui/packages/main/src/types/matplot.tsx

```diff
@@ -58,29 +46,46 @@ function MatplotlibFigureValueFigure(props: MatplotlibFigureValueFigureProps) {
         scale: scale,
         translate: {
           x: (width - spec.width) / 2,
-          y: (scaledHeight - spec.height) / 2
-        }
-      })
+          y: (scaledHeight - spec.height) / 2,
+        },
+      });
 
       hasFigureRendered.current = true;
     }
   }, [width, height, drawFigure, figureId, specProp]);
 
-  return <div ref={figDivRef} style={{ width: "100%", height: `${scaledHeight}px` }} >
-    <div id={figureId} style={{
-      width: "100%", height: '100%',
-      transform: `scale(${scaleAndTranslate.scale})` +
-        ` translate(${scaleAndTranslate.translate.x}px, ${scaleAndTranslate.translate.y}px)`
-    }} />
-  </div>
+  return (
+    <div ref={figDivRef} style={{ width: "100%", height: `${scaledHeight}px` }}>
+      <div
+        id={figureId}
+        style={{
+          width: "100%",
+          height: "100%",
+          transform:
+            `scale(${scaleAndTranslate.scale})` +
+            ` translate(${scaleAndTranslate.translate.x}px, ${scaleAndTranslate.translate.y}px)`,
+        }}
+      />
+    </div>
+  );
 }
 
+// This component can be entirely removed and replaced with ImageValueView
+// a couple of releases after 0.28.0
```

Comment:
It sounds like this could use some bureaucracy!

Please:
- create a new issue describing the intent of the deprecation and the minimum version when it can be done
- add code comments with the issue number where code needs to be deleted in the deprecation
- add a new section in `versions.py` mentioning the deprecation issue can be started after the decided issue

Future you will thank you.

Reply:
Ugh 😉